### PR TITLE
Resurrecting LAMMPS Experiment

### PR DESCRIPTION
**Pull request by csegarragonz**
In this PR I include the plots for LAMMPS, and move the base image to inherit from `faasm:cli`.

### CODE REVIEW

File: aks/deployment.yaml [hunk 1]
```diff
@@ -17,8 +17,8 @@ spec:
       containers:
         - name: mpi-worker
           image: ${IMAGE_NAME}
-          imagePullPolicy: IfNotPresent
-          #imagePullPolicy: Always
+          imagePullPolicy: Always
+          #imagePullPolicy: IfNotPresent
```
Comment (csegarragonz) [inline aks/deployment.yaml:21]: I leave this option commented out, as it is useful to reset the cluster.

File: .gitignore [hunk 2]
```diff
@@ -1 +1,2 @@
 venv
+__pycache__
```
Comment (Shillaker) [inline .gitignore:2]: We could probably put a "proper" Python `.gitignore` in here: https://github.com/github/gitignore/blob/master/Python.gitignore

**Review by Shillaker — APPROVED** [review]
Nice one, looking good. Small comment re. `.gitignore`

File: .gitignore (hunk 2 above)
Comment (csegarragonz) [reply .gitignore:2]: @Shillaker should we copy and paste this here?

Comment (Shillaker) [reply .gitignore:2]: Yeah just copy-paste. Maybe remove some of the bits we know we'll never need but it's not a big deal

**Pull request merged**